spec/domain/sac_imports/membership_years_report_spec.rb

```diff
@@ -55,10 +55,10 @@
 
     expect(csv_report.size).to eq(10)
     expect(csv_report.first).to eq(report_headers)
-    expect(csv_report.second).to eq(["513544", "Müller Hans", "42", nil, nil, "Person not found in hitobito"])
-    expect(csv_report.third).to eq(["513546", "Meier Ursula", "3", "0.0", "3.0", nil])
-    expect(csv_report.fourth).to eq(["513549", "Schneider Peter", "24", "24.0", "0.0", nil])
-    expect(csv_report.fifth).to eq(["513550", "Weber Anna", nil, "14.0", "-14.0", nil])
+    expect(csv_report.second).to eq(["513544", "Müller", "42", nil, nil, "Person not found in hitobito"])
```

Comment:
warum fällt dann hier der vorname weg? war das ein wunsch vom SAC?

Reply:
In NAV1 gibt es keine Spalte mehr mit Vor- und Nachnamen. Damit wir deshalb nicht in CsvSource weitere Logik eingebaut werden muss, um zwei Spalten auf ein Hash Attribut zu mappen, habe ich `navision_name` angepasst:

```
navision_name: "Surname",
```